src/teleop/gui/src/leaflet-rotatedmarker.js

```diff
@@ -0,0 +1,59 @@
+import L from 'leaflet'
```

Comment:
Same here, not sure this file is necessary (also seems like old legacy code that could maybe use some additional commenting/documentation)